docs/devel/kubectl-conventions.md

```diff
@@ -253,6 +254,18 @@ func (g *NamespaceGeneratorV1) validate() error {
 The generator struct (`NamespaceGeneratorV1`) holds the necessary fields for namespace generation. It also satisfies the `kubectl.StructuredGenerator` interface by implementing the `StructuredGenerate() (runtime.Object, error)` method which configures the generated namespace that callers of the generator (`kubectl create namespace` in our case) need to create.
 * `--dry-run` should output the resource that would be created, without creating it.
 
+## Create commands
+
+`kubectl create <resource>` commands fill the gap between "I want to try kube, but I don't know or care what gets created" (`kubectl run`) and "I want to create exactly this" (author yaml and run `kubectl create -f`).
```

Comment:
I've been thinking about  `kubectl create container`, which could create a pod with one container, but if kubectl could suggest `Try 'kubectl create pod' instead`, I'd be fine with that. I only bring this up because "container" isn't a resource.